src/components/schedule-page/import-buttons/import-buttons.component.tsx

```diff
@@ -71,9 +72,11 @@ export function ImportButtonsComponent(): JSX.Element {
           }}
         >
           <ButtonGroup orientation="vertical">
-            <Button component="label">
+            <Button onClick={() => fileUpload.current?.click()}>
               Wczytaj
               <input
+                ref={fileUpload}
+                id="file-input"
                 data-cy="file-input"
                 onChange={(event): void => handleImport(event)}
                 style={{ display: "none" }}
```

Comment:
@Ehevi jeżeli  szybko ci pójdzie ze stylowaniem, to myślę, że mogłabyś zająć się utworzeniem komponentu dropdown'a. Ale to przegadamy już jak skończysz ten task :)

Reply:
@pectom , powiedz proszę też co o tym sądzisz?

Reply:
Super pomysł @Ehevi zostanie naszą ekspert od dropdownów XD. Przegadaliśmy już wstępnie z @Qwebeck jakby to miało wyglądać. Wyróżnimy 2 zadania - jedno na dodanie komponentu, a drugie na dołączenie styli.